README.md

```diff
@@ -6,11 +6,34 @@ See also https://github.com/tulibraries/centralized-metadata-dags.
 
 ## System Requirements 
 
-TO DO
+Posgres DB: latet
```

Comment:
@dkinzer -- just a minor mistyping here.

Reply:
Thanks @ebtoner. I made the change requested.